user-service/src/main/resources/secrets.yml

```
@@ -0,0 +1,5 @@
+secrets:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/oakcan_db
+    password: 12345
+    username: postgres
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)